app/src/main/cpp/skyline/input/touch.cpp

```diff
@@ -15,28 +15,34 @@ namespace skyline::input {
         }
     }
 
-    void TouchManager::SetState(const span<TouchScreenPoint> &points) {
+    void TouchManager::SetState(const span<TouchScreenPoint> touchPoints) {
```

Comment:
```suggestion
    void TouchManager::SetState(span<TouchScreenPoint> touchPoints) {
```
We don't use const generally